### PR TITLE
feature/footer

### LINKED ISSUE #45: create a footer
create a footer as different organizations similar to ours

### PR DESCRIPTION
**Pull request by kasin-it**
resolves #45

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+				{ label: 'About Us', href: '#' },
+				{ label: 'Contact', href: '#' },
+				{ label: 'Services', href: '#' },
+				{ label: 'FAQ', href: '#' },
+			],
+		},
+	];
+
+	return (
+		<div className="w-full flex justify-center py-10">
```
Comment (Kamil-Drozdz) [inline src/components/Footer.tsx:78]: we can put here more semantic tag

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import Link from 'next/link';
+import { Navigation } from '@/enums/navigation.enum';
+import { SOCIAL_PROVIDER_REACT_ICONS } from '@/constants/social-provider-react-icons';
+
+type FooterSectionType = {
```
Comment (Kamil-Drozdz) [inline src/components/Footer.tsx]: in the project we usually used interface, I think there is type in several places but is there any special purpose with type?

File: src/components/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+		</ul>
+	</div>
+);
+
+const SocialIcons = () => (
+		<div className="inline-flex gap-x-5 text-2xl">
+				{SOCIAL_PROVIDER_REACT_ICONS.map(provider => (
+
+					<Link
+					key={provider.provider}
```
Comment (Kamil-Drozdz) [inline src/components/Footer.tsx:33]: sometimes you use index and sometimes names

File: src/components/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+							<p className="text-sm">© 2023 Coders-Factory</p>
+						</div>
+
+						<SocialIcons />
+					</div>
+				</div>
+
+				{FOOTER_NAVIGATION.map((section, index) => (
+					<FooterSection
+						key={index}
```
Comment (Kamil-Drozdz) [inline src/components/Footer.tsx]: here

File: src/components/Footer.tsx (hunk 3 above)
Comment (Kamil-Drozdz) [reply src/components/Footer.tsx:33]: if there are unique names and no more will be needed, it is better to use the name in this context

File: src/components/Footer.tsx (hunk 1 above)
Comment (Kamil-Drozdz) [inline src/components/Footer.tsx]: I Think we can put footer as container for that

File: src/components/Footer.tsx (hunk 2 above)
Comment (kasin-it) [reply src/components/Footer.tsx]: Not, really

File: src/constants/social-provider-react-icons.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import { SocialsLinks } from "@/enums/socials.enum";
+import { BiLogoLinkedinSquare } from "react-icons/bi";
```
Comment (Kamil-Drozdz) [inline src/constants/social-provider-react-icons.tsx]: remove unnecessary imported icons, some are not needed?

**Review by Kamil-Drozdz — APPROVED** [review]
Aprrove

**Pull request merged**